map_creator/src/load_reachability_map.cpp

```diff
@@ -33,71 +24,42 @@ int main(int argc, char **argv)
 
     int count = 0;
 
-    time_t startit, finish;
-    time(&startit);
-    const char *FILE = argv[1];
-    hid_t file, poses_group, poses_dataset, sphere_group, sphere_dataset, attr;
-    file = H5Fopen(FILE, H5F_ACC_RDONLY, H5P_DEFAULT);
-
-    // Poses dataset
-
-    poses_group = H5Gopen(file, POSE_GROUPNAME, H5P_DEFAULT);
-    poses_dataset = H5Dopen(poses_group, POSES_DATASETNAME, H5P_DEFAULT);
-
-    std::multimap< std::vector< double >, std::vector< double > > PoseColFilter;
-    hdf5_dataset::Hdf5Dataset hd5;
-    hd5.h5ToMultiMapPoses(poses_dataset, PoseColFilter);
+    hdf5_dataset::Hdf5Dataset h5(argv[1]);
+    h5.open();
 
-    // Sphere dataset
-    sphere_group = H5Gopen(file, SPHERE_GROUPNAME, H5P_DEFAULT);
-    sphere_dataset = H5Dopen(sphere_group, SPHERE_DATASETNAME, H5P_DEFAULT);
-
-    std::multimap< std::vector< double >, double > SphereCol;
-    hd5.h5ToMultiMapSpheres(sphere_dataset, SphereCol);
-
-    // Resolution Attribute
+    MultiMap PoseColFilter;
```

Comment:
@akgoins All the namespaces are corrected and CamelCased
variables are lower_case_under_scored
SpereCol is resized is base_placement_widget

Please check again and merge the PR